frontend/src/App.vue

```diff
@@ -54,16 +53,67 @@ export default {
       );
     },
   },
-  watch: {
-  },
+  watch: {},
   methods: {
     async onAction(contextualItem, index) {
       const itemFqid = contextualItem?.item?.fqid;
```

Comment:
`itemFqid` could be `undefined`
should the action still be performed in this case?